src/main/java/com/zufar/onlinestore/cart/entity/ShoppingSessionItem.java

```diff
@@ -0,0 +1,70 @@
+package com.zufar.onlinestore.cart.entity;
+
+import com.zufar.onlinestore.product.entity.ProductInfo;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Objects;
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Table(name = "shopping_session_item")
+public class ShoppingSessionItem {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    private UUID id;
+
+    @ManyToOne
```

Comment:
It it better to put @ManyToOne(fetch = FetchType.LAZY)

Reply:
If we want to get all items associated with shopping session using ShoppingSessionItem we can make a query like this:
ShoppingSessionItemRepository.findAllByShoppingSession(shoppingSession) and also we will get all ShoppingSessionItem objects without extra **shoppingSession** because of fetch = FetchType.LAZY

Reply:
Item without product does not make any sense